scripts/pre_translate.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-INASAFE_DEV_PATH=$HOME/dev/python/inasafe-dev/
+INASAFE_DEV_PATH=$HOME/dev/python/inasafe/
```

Comment:
I don't think we need any of these python setup lines anymore...or QGIS_PREFIX_PATH...